src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_rpc_plugin.h

```diff
@@ -42,11 +42,14 @@
 #include "rc_rpc_plugin/rc_capabilities_manager.h"
 #include "rc_rpc_plugin/rc_consent_manager.h"
 #include "rc_rpc_plugin/resource_allocation_manager.h"
+#include "utils/ilogger.h"
```

Comment:
Do we need both forward declaration and include?

Reply:
no